README.md

```diff
@@ -11,11 +11,14 @@ The source code is released under the MIT License.
     pip install python-rake
 
 ### Usage ###
+#for external .txt files
     import RAKE
-    Rake = RAKE.Rake([path_to_your_stopwords_file]);
-    # You can use one of the stoplists included in the repository under stoplists/
+    Rake = RAKE.Rake(<path_to_your_stopwords_file>);
+    Rake.run(text);
+#for built in stoplists
+    import RAKE
+    Rake = RAKE.Rake(<keyword>); #current keywords are SmartStopList and FoxStopList
```

Comment:
In retrospect there's a much better way to do it as RAKE.SmartStopList() that I didn't think of. Changing now.